nautobot/core/tables.py

```diff
@@ -439,6 +439,95 @@ def header(self):  # pylint: disable=invalid-overridden-method
         return ""
 
 
+class ApprovalButtonsColumn(django_tables2.TemplateColumn):
+    """
+    Render approve, deny, and comment buttons for an approval workflow stage instance.
+
+    :param model: Model class to use for calculating URL view names
+    :param prepend_template: Additional template content to render in the column (optional)
+    :param return_url_extra: String to append to the return URL (e.g. for specifying a tab) (optional)
+    """
+
+    buttons = ("detail", "changelog", "approve", "deny", "comment")
+    attrs = {"td": {"class": "text-right text-nowrap noprint"}}
+    # Note that braces are escaped to allow for string formatting prior to template rendering
+    template_code = """
```

Comment:
Any chance we could do this as a standalone template file to somehow avoid the need for string formatting and make it easier to read?